plugins/modules/sonic_aaa.py

```diff
@@ -163,6 +213,18 @@
         login_auth_method:
           - local
           - ldap
+      accounting:
+        commands_accounting:
+          accounting_method:
+            - tacacs+
+            - logging
+          accounting_record_type: 'START_STOP'
```

Comment:
Please update all record type references in the examples with 'start-stop' and 'stop-only'.